rest/codegen/paths.go

```diff
@@ -52,17 +93,20 @@ type (
 
 var pathTmpl = template.Must(template.New("path").
 	Funcs(template.FuncMap{
-		"join": strings.Join,
-		"add":  codegen.Add,
+		"add":       codegen.Add,
+		"goTypeRef": codegen.GoTypeRef,
 	}).
 	Parse(pathT))
 
 // PathWriter returns the path generators writer.
-func PathWriter(r *design.RootExpr) codegen.FileWriter {
+func PathWriter(api *goadesign.APIExpr, r *design.RootExpr) codegen.FileWriter {
+	title := fmt.Sprintf("%s HTTP request path constructors", api.Name)
 	sections := []*codegen.Section{
-		codegen.Header("", "http", []*codegen.ImportSpec{
-			{Path: "context"},
+		codegen.Header(title, "http", []*codegen.ImportSpec{
```

Comment:
Yes `v1` already does that and `v2` will as well. We cannot use `goimport` directly because we don't want to add imports (the added imports are not guaranteed to be correct). However there is code that removes extra imports.